beacon-chain/sync/validate_beacon_attestation.go

```diff
@@ -71,6 +70,32 @@ func (s *Service) validateCommitteeIndexBeaconAttestation(ctx context.Context, p
 	if data.Slot == 0 {
 		return pubsub.ValidationIgnore, nil
 	}
+
+	preState, err := s.cfg.chain.AttestationTargetState(ctx, data.Target)
```

Comment:
Why are we calling this here ? There must be strong justification for 
- fetching a state
- fetching the committee from it

As these are additional bottlenecks on our highest traffic gossip path

Reply:
Also why is this being called before the current gossip checks